projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -49,6 +58,14 @@ export class PageTimeRangePreferenceService {
     );
   }
 
+  public tenantTimeRangeMapHasValue(tenantDefaultTimeRangeMap: FeatureFlagValue, rootLevelPath: string): boolean {
```

Comment:
should already be typed coming in here. Change above to avoid casts: `this.featureStateResolver.getFeatureFlagValue<Dictionary<string>>(ApplicationFeature.TenantDefaultTimeRangeMap)` . Then can just do `rootLevelPath in tenantDefaultTimeRangeMap` since we don't really care about the empty string case (I'd probably inline it, but if you prefer you can leave the function).